docs/docs/risingwave.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "RisingWave"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# RisingWave
+
+[RisingWave](https://risingwave.com/) is a Postgres-compatible SQL database designed for real-time event streaming data processing, analysis, and management. It can ingest millions of events per second, continuously join and analyze live data streams with historical tables, serve ad-hoc queries in real-time, and deliver fresh, consistent results.
+
+## Supported Features
+
+RisingWave supports batch reads and streaming writes of Iceberg tables via its built-in source and sink connectors. For more information, see the [Iceberg source connector documentation](https://docs.risingwave.com/integrations/sources/apache-iceberg) and [Iceberg sink connector documentation](https://docs.risingwave.com/integrations/destinations/apache-iceberg).
+
+## Table Formats and Warehouse Locations
+
+Currently, RisingWave only supports the Iceberg V2 table format and S3-compatible object storage as Iceberg warehouse locations.
+
+## Catalogs
+
+RisingWave supports the following catalogs:
+
+- `rest`
+- `jdbc`
```

Comment:
This is only called JDBC in Java, in PyIceberg and Iceberg-Rust it is called the `sql` catalog. This reminds us that naming things is one of the hardest things in computer science.

```suggestion
- `jdbc` / `sql`
```

Considering that Risingwave is Postgres compatible, you could also use the SQL Catalog backed by Risingwave itself :)

Reply:
> This is only called JDBC in Java, in PyIceberg and Iceberg-Rust it is called the `sql` catalog. This reminds us that naming things is one of the hardest things in computer science.
> 
> Considering that Risingwave is Postgres compatible, you could also use the SQL Catalog backed by Risingwave itself :)

I echoed the point that naming things is one of the hardest things. :) 
Much appreciate your suggested change.